src/components/diagrams/diagram.js

```diff
@@ -95,6 +96,23 @@ const Diagram = (props) => {
         dispatch(decrementNetworkAreaDiagramDepth());
     };
 
+    const showAlert = () => {
+        const { hasNonVoltageLevelOrSubstation, disabled, svgType } = props;
+        const isVoltageLevel = DiagramType.VOLTAGE_LEVEL === svgType;
+        const message = isVoltageLevel
+            ? 'VoltageLevelNotFound'
+            : 'SubstationNotFound';
+        return (
+            <Box position="relative" left={0} right={0} top={0}>
+                {hasNonVoltageLevelOrSubstation && !disabled ? (
+                    <AlertCustomMessageNode message={message} noMargin={true} />
+                ) : (
+                    <AlertInvalidNode noMargin={true} />
```

Comment:
Maybe this should be changed if we use AlertInvalidNode with a new "message" props instead of the new component ? (see below)

Reply:
I already thought about it but the name "AlertInvalidNode"  is not adequate so I created another component.